Split keyword operators in a separate category

planetis-m opened the pull request:
This splits and|as|div|in|is|isnot|mod|notin|of|or|shl|shr|xor from the rest. It is done in order to have more highlighting options. Might be technically [wrong](https://nim-lang.org/docs/manual.html#lexical-analysis-operators). Python highlighting [file](https://raw.githubusercontent.com/KDE/syntax-highlighting/master/data/syntax/python.xml)

planetis-m committed:
Split bool/binary operators in a separate category

planetis-m commented:
`var` and `type` can be both parameter attributes and keywords, {and similarly (`object`, `enum`, `tuple` are keywords and types), (`of` control flow and keyword)} can all this be supported?
Also echo, sizeof, etc could be split in category called builtin_funcs. What do you say?

planetis-m committed:
Imitate what python hl uses for default colors

b3liever commented:
For the last commit see this: https://raw.githubusercontent.com/KDE/syntax-highlighting/master/data/syntax/python.xml

planetis-m committed:
More changes

 - Add builtins category
 -  Use better default styles
 -  Use worddetect instead of regexpr

planetis-m commented:
I broke procdefs for some reason...

planetis-m commented:
@ryukoposting pls take a look
